app/controllers/application_controller.rb

```diff
@@ -1,5 +1,58 @@
+require 'csv'
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  attr_reader :jobs, 
+              :quotes,
+              :jobs_by_location,
+              :jobs_by_interest
+
+  def dashboard
```

Comment:
also - this method is doing a ton - how can we refactor this (maybe using a PORO) to cut down on the lines of code?